pkg/operator/reports.go

```diff
@@ -734,6 +740,65 @@ func (op *Reporting) runReport(logger log.FieldLogger, report *metering.Report)
 	return nil
 }
 
+func deleteReport(logger log.FieldLogger, report *metering.Report, op *Reporting) {
+	if reportNotUsedAsInput(logger, report, op) {
+		logger.Infof("Deleted report after retention period: %s, past reportingEnd: %s",
+			report.Spec.Expiration, report.Spec.ReportingEnd)
+	}
+}
```

Comment:
We probably want to return at least an error here, and use the metering clientset to delete a list of Report custom resources that exist within a namespace.

Reply:
yeah didn't actually put in the delete action yet 🍅

Reply:
okay, delete action in place now: 
in 099029d

Reply:
Errors about problems in this new check-if-report-expired process are logged, but was thinking we don't want to stop the calling func, `runReport` from doing its thing just because we're unable to delete the report?